packages/types/src/block-kit/elements.ts

```diff
@@ -0,0 +1,224 @@
+// This file contains objects documented here: https://api.slack.com/reference/block-kit/block-elements
```

Comment:
Should this just be called block-elements.ts as well based on the documentation? Would line up better alphabetically with blocks.ts and makes it clear they belong to blocks

Reply:
Yes excellent point! Will do